assets/src/editor/blocks/landing-page-hero/index.js

```diff
@@ -205,9 +255,15 @@ export const settings = {
 							tagName="h1"
 							value={ title }
 						/>
+						<RichText.Content
```

Comment:
Since this is a new attribute, there needs to be a graceful fallback in case it isn't set. Can you wrap in a check to see that `description` exists before outputting it?